packages/frontend/src/components/FlowStep/index.tsx

```diff
@@ -40,6 +40,7 @@ type FlowStepProps = {
   onContinue?: () => void
 }
 
+// FIXME (ogp-weeloong): remove this; not needed since we already do validation in FlowSubstep.
```

Comment:
question: is this not necessary because we don't have fields that use `dependsOn`? not really sure how you tell this is extra validation

Reply:
we don't use `dependsOn`, and we already have validation for `required` in `FlowSubStep`'s `validateSubstep` function